src/GoStoneGroups.ts

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2012-2020 Online-Go.com
```

Comment:
Across the board... nice 😏

Reply:
:D :D

Reply:
Totally true 😂 and I know a lot of projects don't even bother with dates on the (c) notice - maybe something we can consider 🙃

Reply:
I looked at what some higher profile projects do in this regard, here's what I found:

* React, Node.js both have the header but no dates
* Angular, D3.js, Babylon.js, Rust don't bother with headers

None used dates. So I'm inclined to agree, I think we can ditch the dates. Since some of the files are under APL and some under AGPL, I'm thinking maybe the header makes sense to be clear, but I could be pretty easily convinced to remove the header from everything that's AGPL since that's kinda the default for the repo, and just have the APL header for those few files that are APL.

Reply:
Oh nice! Re headers vs no headers, I'd selfishly be happy with no headers because I don't really have a reason to look at the legal boilerplate. But I think there are good reasons to keep the headers as well.  I've heard the argument that if someone wants to borrow one file from the repo, it's easier to make sure the copyright/license makes it over as well.